.github/dependabot.yml

```diff
@@ -21,7 +21,7 @@ updates:
 - package-ecosystem: composer
   directory: "/"
   schedule:
-    interval: daily
+    interval: weekly
```

Comment:
I think it's better to be weekly, so we focus on it once a week :)

Reply:
I was ok with daily, but ok

Reply:
also linked a bit to the rythme we do releases 😅 not worth having it daily IMO
weekly looks to be a good compromise